DevCycle/Models/DevCycleEvent.swift

```diff
@@ -32,27 +32,27 @@ public class DevCycleEvent {
             self.event = DevCycleEvent(type: nil, target: nil, clientDate: nil, value: nil, metaData: nil)
         }
         
-        public func type(_ type: String) -> EventBuilder {
+        public func type(_ type: String?) -> EventBuilder {
```

Comment:
why are we changing the public type of all these? like why would we want users to be able to set nullable on the public EventBuilder?

Reply:
the stored types on this class are all already optional, the only type that is required is type and it gets checked when `build` gets called